rl/boltzmann_money/README.md

```diff
@@ -0,0 +1,15 @@
+# Balancing Wealth Inequality
+This folder showcases how to solve the Boltzmann wealth model with Proximal Policy Optimization (PPO) from Stable Baselines.
+
+## Key features:
+
+- Boltzmann Wealth Model: Agents with varying wealth navigate a grid, aiming to minimize inequality measured by the Gini coefficient.
+- PPO Training: A PPO agent is trained to achieve this goal, receiving sparse rewards based on Gini coefficient improvement and a large terminal reward for achieving low inequality.
+- Mesa Data Collection and Visualization: The Mesa data collector tool tracks Gini values during training, allowing for real-time visualization.
+- Visualization Script: Visualize the trained agent's behavior with Mesa's visualization tools, presenting agent movement and Gini values within the grid. You can run `server.py` file to test it with pre-trained model.
+
+## Model Behaviour
+The adgent as seen below learns to move towards a corner of the grid. These brings all the agents together allowing exchange of money between them resulting in reward maximization.
```

Comment:
The result is a bit sus to me. How could all agents simultaneously decide to go to 1 corner? This implies they all use the same weight, which is biased toward the top left. They should have instead look for nearby agents, and seek to get closer to their neighbors, until they are all in the same cell.

Reply:
In nowhere in the readme can I find any indication of such fine print. It needs an explicit disclaimer that the behavior is not what ideally we should expect: where the agents seek the other agents.

Reply:
Done